Mathlib/Analysis/RCLike/Inner.lean

```diff
@@ -0,0 +1,150 @@
+/-
+Copyright (c) 2023 Yaël Dilies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dilies
+-/
+import Mathlib.Analysis.InnerProductSpace.PiL2
+
+/-!
+# L2 inner product of finite sequences
+
+This file defines the weighted L2 inner product of functions `f g : ι → R` where `ι` is a fintype as
+`∑ i, conj (f i) * g i`. This convention (conjugation on the left) matches the inner product coming
+from `RCLike.innerProductSpace`.
```

Comment:
The module docstring needs updating now.

Reply:
Does it? It seems up to date. `RCLike.innerProductSpace` still has its conjugation on the left, no?

Reply:
There's no weight in that sum.

Reply:
Sure, but I'm not saying that they are equal. I'm saying they are using the same convention when it comes to picking a side in which the inner product is antilinear.

Reply:
This reads to me as if you are saying they are equal: 
"This file defines the weighted L2 inner product of functions `f g : ι → R` where `ι` is a fintype as
`∑ i, conj (f i) * g i`."
As Jireh says, this sentence says you're defining the weighted inner product, but there's no weighting here